pkg/syncer/types.go

```diff
@@ -45,6 +45,9 @@ const (
 	// key for HealthStatus annotation on PVC
 	annVolumeHealth = "volumehealth.storage.kubernetes.io/health"
 
+	// key for PV to vVol mapping annotation on PVC
+	annPVtoVVOL = "cns.vmware.com"
```

Comment:
using such a general key is not a good idea. Please try to name it more specifically. e.g., `csi.vsphere.pv-to-vvol-mapping`

Reply:
As we discussed in the design spec review the annotation should be in format: "cns.vmware.com/<pvuuid>":<backingDiskObjectId>
we use "cns.vmware.com/<pvuuid>" as the key, and the annotation is set as:
metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annPVtoVVOL+ "/" + uuid, backingObjectId)

Reply:
I don't believe put PV "uuid" in annotation key is the right thing to do. 
Instead, it might better to set `annPVtoVVOL` to "cns.vmware.com/pv-to-vvol-mapping". And moving the PV uuid is to annotation value. 
So the PVC annotation will be like,
cns.vmware.com/pv-to-vvol-mapping=<pv uuid>:<vvol backing object id>.

Additionally, pv uuid might not be necessary, as it is available in VolumeHandle in the bound PV of each PVC. I will leave it to you.